test/Interop/Cxx/union/Inputs/anonymous-union-partly-invalid.h

```diff
@@ -0,0 +1,16 @@
+@class C;
+@interface C
+{}
+@end
+
+struct S {
+  union {
+    C *t;
+    char c;
+  };
+  S(const S &s) {}
+  ~S() { }
```

Comment:
Additionally, it's odd that this doesn't fail when the special members are gone, this contributes to my thinking that we don't know the whole story here.

Reply:
Ah yeah I agreed. I thought there was something going on where the special members of the containing struct affected the anon-union's. If it is ok with you to land this so I can be unblocked, I will continue investigating in the background if that is ok?